pokete_classes/input.py

```diff
@@ -27,7 +27,8 @@ def text_input(obj, _map, name, ev, wrap_len, max_len=1000000):
             obj.rechar(hard_liner(wrap_len, name + "█"))
             _map.show()
             ev.clear()
-        elif ev.get() not in ["", "Key.enter", "exit", "Key.backspace", "Key.shift",
+        elif ev.get() not in ["", "Key.enter", "exit", "Key.backspace",
+                              "Key.shift",
                         "Key.shift_r", "Key.esc"] and len(name) < max_len:
             if ev.get() == "Key.space":
```

Comment:
```suggestion
        elif ev.get() not in ["", "Key.enter", "exit", "Key.backspace",
                              "Key.shift",
                              "Key.shift_r", "Key.esc"] and len(name) < max_len:
```

Reply:
where did the `if ev.get() == "Key.space":` go?